### DISCUSSION TITLE
Document some more items like renderScene

### DISCUSSION THREAD

**Discussion by afkcodes** [grahammendick/navigation#764] [General]
Hey @grahammendick , I was actually going through the medley app and i found that renderScene is use there but there is no documentation for the same. Can we slowly start documenting those. 

And what does the renderScene do. and in which cases we should use this.

**Comment by grahammendick** [comment]
Hey, before [I added a component api](https://github.com/grahammendick/navigation/pull/592) a couple of years ago `renderScene` was the only way to use the Navigation router. But now there’s a component api, I don’t advertise the `renderScene` approach anymore.

I added the component api in a non-breaking way, so the `renderScene` still works exactly the same and there was no need to update the samples. It's useful to keep a sample using `renderScene` so that inquisitive people can find out about it without cluttering up the documentation.
